force-app/main/default/lwc/donationHistoryTable/__tests__/donationHistoryTable.test.js

```diff
@@ -1,7 +1,13 @@
 import { createElement } from 'lwc';
 import DonationHistoryTable from '../donationHistoryTable';
+import { extended, full } from "@sa11y/preset-rules";
+import { setup } from "@sa11y/jest";
 const MOCK_DATA = require('./data/donationHistoryTableData.json');
 
+beforeAll(() => {
```

Comment:
This function is causing the existing Sa11y test suite to fail with the following error message.
`Cannot add a hook after tests have started running. Hooks must be defined synchronously.`

Reply:
Hi Andrew. Thanks for the suggestion. This has now been fixed. Please, re-review.

Reply:
I'm still seeing the same issue in this test suite: `Cannot add a hook after tests have started running. Hooks must be defined synchronously.`

Reply:
@lparrott You are right. I had solved the issue for Giving Summary component, but not for Donation History table component. It should be working now